independent-projects/arc/processor/src/test/java/io/quarkus/arc/processor/TypesTest.java

```diff
@@ -93,6 +94,17 @@ public void testGetTypeClosure() throws IOException {
         assertDoesNotThrow(
                 () -> verifyEagleTypes(Types.getClassBeanTypeClosure(index.getClassByName(DotName.createSimple(Eagle.class)),
                         dummyDeployment)));
+
+        // raw type bean
+        Set<Type> rawBeanTypes = Types.getClassBeanTypeClosure(index.getClassByName(DotName.createSimple(MyRawBean.class)),
+                dummyDeployment);
+        assertEquals(rawBeanTypes.size(), 5);
+        assertTrue(rawBeanTypes.contains(Type.create(DotName.createSimple(MyRawBean.class.getName()), Kind.CLASS)));
```

Comment:
Just nitpicking but this looks like a good candidate for a util method...?
```java
private void assertContainsType(Class<?> clazz, Set<Type> rawBeanTypes) {
   assertTrue(rawBeanTypes.contains(Type.create(DotName.createSimple(clazz.getName()), Kind.CLASS)));
}
```

Reply:
Fair enough, introduced util method :)